### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by ger619**
For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.

You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.

After that, you could implement the popup window.

### CODE REVIEW

File: style.js [hunk 1]
```diff
@@ -1,21 +1,179 @@
...
+const cards = [{
+
+  id: 1,
+  title: 'Tonic',
+  imageM: './images/Portfolio.png',
+  imageD: './images/Portfolio1-big.png',
+  heading2: ['Canopy', 'Back End Dev', '2015'],
+  cardDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  languages: ['html', 'css', 'javascript'],
+
```
Comment (micheaol) [inline style.js:30]: Please ensure that the object contains at least the following information about the project: name, description, featured image, technologies, `link to live version`, and `link to the project source code`.

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

File: style.js [hunk 2]
```diff
@@ -1,21 +1,179 @@
...
+                    <div>
+                        <img class="card" src="${card.imageM}" alt="Portfolio1" />
+                        <img class="animated" src="${card.imageD}" alt="nature-animated-image">
+                    </div>
+                    <div class="cards74">
+                        <h2 class="tonic-class-${index}">${card.title}</h2>
+                        
+                        <h5 class="heading-canopy"><b>CANOPY</b> <span>·</span><small> Back End Dev <span>·</span> 2015 </small> </h5>
+                        <p class="description">
+                           ${card.cardDescription}
```
Comment (micheaol) [inline style.js:75]: Please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, margin)

Your Implementation | Figma Design
:---:|:---:
image | image

## Mobile version:
Please ensure that the popup is responsive, it fits mobile and desktop sizes properly. Please ensure that a blurry effect is visible in your design like the Figma design. 

Your Implementation | Figma Design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @ger619 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.js (hunk 2 above)
Comment (ger619) [reply style.js:75]: Done

**Review by Whoistolu — COMMENTED** [review]
Hi @ger619 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that whose LINTERS are failing. Kindly ensure that all the linter checks turn green before requesting another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**